calibration/CTDMOG/CGINS-CTDMOG-13645__20150625.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13645,CC_ptempa0,154.1158,
 37-13645,CC_ptempa1,-0.05393076,
 37-13645,CC_ptempa2,-3.256799e-06,
-37-13645,CC_wbotc,3.7924e-07,
+37-13645,CC_wbotc,3.792352e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT